src/main/java/sirius/kernel/xml/Outcall.java

```diff
@@ -823,4 +810,14 @@ public static Duration getDefaultConnectTimeout() {
     public static Duration getDefaultReadTimeout() {
         return defaultReadTimeout;
     }
+
+    /**
+     * Sets a custom blacklist id.
+     * The default blacklist id is the host of the uri.
+     *
+     * @param blacklistId The custom blacklist id to set.
+     */
+    public void setBlacklistId(@Nonnull String blacklistId) {
```

Comment:
I would prefer a builder pattern, so you can simply use `new Outcall(...).withBlacklistId(...)`, avoiding the need for extra constructors.

Reply:
The setter is needed for things like the XMLCall where the Outcall is created in the constructor and cannot be replaced and can only be accessed via a getter. The constructor with the blacklistid is removed and not needed anymore.